locales/en/home.json

```diff
@@ -4,6 +4,7 @@
   "hero-p": "Bring your inner creativity with Vignette, the open source modular streaming toolkit for virtual streaming.",
   "join-discord-long": "Join our Discord for updates",
   "join-discord-short": "Join our Discord",
+  "join-survey": "Join our Survey",
```

Comment:
This sounds too invasive, maybe something more inviting so we can invite them to take the survey like "Can't wait? Let's get to know each other!"